client/models/channel.coffee

```diff
@@ -4,3 +4,26 @@ sd = require('sharify').data
 module.exports = class Channel extends Backbone.Model
 
   urlRoot: "#{sd.API_URL}/channels"
+
+  getFeatures: ->
+    switch @get 'type'
+      when 'editorial' then {
+        features: ['header', 'superArticle']
+        sections: ['text', 'artworks', 'images', 'image_set', 'video', 'embed', 'callout', 'toc']
+        associations: ['artworks', 'artists', 'shows', 'fairs', 'partners','auctions']
+      }
+      when 'team' then {
+        features: []
+        sections: ['text', 'artworks', 'images', 'image_set', 'video', 'embed', 'callout']
+        associations: []
+      }
+      when 'support' then {
+        features: []
+        sections: ['text', 'artworks', 'images', 'video', 'callout']
+        associations: ['artworks', 'artists', 'shows', 'fairs', 'partners','auctions']
+      }
+      when 'partner' then {
+        features: []
+        sections: ['text','artworks', 'images','video']
+        associations: []
+      }
```

Comment:
I think this is a great idea! Keeping them separate is useful at the data-level to avoid redundant data and "syncing" from services—but at the client-level I think it makes sense to join these for the UI to be easier to reason about.